src/popup/basics/index.tsx

```diff
@@ -181,6 +186,23 @@ export const FormTextField = styled(Field)`
   font-size: 1rem;
   padding: 1.875rem 2.25rem;
   width: 100%;
+  resize: none;
+
+  &::-webkit-input-placeholder {
```

Comment:
I noticed placeholder text's font family for `/recover-account`'s `textarea` is different from monospace that user gets when they start typing. Added this to reflect that.